projects/ngx-location-viewer/src/lib/types/mapserver/layerinfo-response/layer-info.model.ts

```diff
@@ -1,9 +1,11 @@
+import { GeometryTypes } from '../../geometry-types.enum';
+
 export interface LayerSpecificInfo {
   id: number;
   name: string;
   defaultVisibility: boolean;
   drawingInfo: LayerDrawingInfo;
-  geometryType: string;
+  geometryType: GeometryTypes;
```

Comment:
is dit geen breaking change in de API van deze component?

Reply:
Dit is niet een model dat gebruikt wordt voor API van deze component.
Maar een model dat de response van ARCGIS laag informatie definieert.